### PR TITLE
CONCF-543 check if article exists and only then add it to API response

### PR DESCRIPTION
**Pull request by bognix**
@vforge

### CODE REVIEW

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php [hunk 1]
```diff
@@ -272,18 +276,23 @@ public function processCuratedContent( $data ) {
...
+	 * If article doesn't exist (Title is null) return null.
+	 * In other case return item with updated article_local_url.
 	 * TODO Implement cache for release version.
 	 * Platform Team is OK with hitting DB for MVP (10-15 wikis)
 	 *
 	 * @param $item
 	 * @return mixed
 	 */
-	private function addArticleUrlToCuratedContentItem( $item ) {
+	private function processItem( $item ) {
```
Comment (rogatty) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: `processCuratedContentItem`

**Comment by rogatty** [comment]
Change name of the method and it LGTM.

**Comment by rogatty** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**